boards/arm/adafruit_trinket_m0/doc/adafruit_trinket_m0.rst

```diff
@@ -0,0 +1,147 @@
+.. _adafruit_trinket_m0:
+
+Adafruit Tinket M0
+##################
+
+Overview
+********
+
+The Adafruit Trinket M0 is a tiny (27 mm x 15 mm) development board
```

Comment:
Maybe add ``ARM`` before ``development board`` (to let folks know the architecture early on)

Reply:
Done